### ISSUE TITLE
Rumble/vibration not working.

### ISSUE THREAD

**Issue by Szperov** [csutorasa/XOutput#62]
Hello, and than you for this awesome program!
Unfortunately on my side the rumble function is not working for some reason.
I have a Playstation SCPH-110 controller, and connected with a cheap eBay Sony to USB converter (recognized as "Twin USB joystick").
Drivers are installed, and under the default windows 10 x64 built in program called "Game controllers/Properties/Vibration test" it's working properly. 
ViGEm is installed, and recognized by xoutput. Under diagnostics force feedback motors count 4. And green mark say okay.
Any idea whats wrong?
![image](https://user-images.githubusercontent.com/43072354/45229534-54285c00-b2c6-11e8-8bfb-aa68d02b0a85.png)

**Comment by csutorasa** [comment]
I only tested force feedback with a pair of motors (small and big motors). You have a "Twin" joystick that means you have 2 pairs of motors. I think this may be the issue.

I will look into this, if I can emulate 4 motors somehow. If I cannot do that I will release a test version.

**Comment by csutorasa** [comment]
I agree on that you have only 2 motors as usual. However the driver states 4, as you can see in the diagnostics.

I cannot promise any ETA, but I will try to work on this.

**Comment by Szperov** [comment]
Thanks, I'll be patient - using this controller for a decade now. :)

**Comment by csutorasa** [comment]
The force feedback debug logging is extended in the newer versions. Can you provide me a log file from the latest version?

**Comment by Yonge714** [comment]
I also have a Twin USB joystick. I have one of the ports on it disabled since I use various controllers through Steam, as well as your program. Does this logfile help?
[XOutput.log](https://github.com/csutorasa/XOutput/files/3561937/XOutput.log)
